nav2_mppi_controller/src/parameters_handler.cpp

```diff
@@ -66,15 +69,21 @@ ParametersHandler::dynamicParamsCallback(
     {
       callback->second(param);
     } else {
-      RCLCPP_WARN(logger_, "Parameter %s not found", param_name.c_str());
+      if (verbose_) {
```

Comment:
I approve of these changes, but I think you can remove all the comments. The log warning content is pretty clear

Reply:
Is there still a question on the second success comment?

Reply:
Removed all comments.

Added test to check behavior when trying to get a parameter that has not been declared. ie you get an exception.

Note that to get the log warning content the handler needs to have verbose_ set to true.  Maybe this code path should always warn.

Reply:
I went a bit further/deeper and used the ParameterType::Dynamic / Static to improve the log output.

Now attempts to set parameters that are not declared will report an error.
There will be a debug message indicating that as static parameters are being ignored if in verbose mode otherwise the log is silent.